src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -47,8 +47,83 @@ internal class RazorSemanticTokensLegend
         public static int MarkupCommentPunctuation => TokenTypesLegend[MarkupCommentPunctuationString];
         public static int MarkupComment => TokenTypesLegend[MarkupCommentString];
 
+        public static int CSharpKeyword => TokenTypesLegend["keyword"];
+        public static int CSharpVariable => TokenTypesLegend["variable"];
+        public static int CSharpOperator => TokenTypesLegend["operator"];
+
         private static readonly IReadOnlyCollection<string> _tokenTypes = new string[] {
```

Comment:
Random note, [this PR](https://github.com/dotnet/roslyn/pull/47960) will add in a new record classification type, so I believe this list may have to be updated once that goes in (I can let you know when it does).

Reply:
As a note for future Ryan, the relevant section is [right here](https://github.com/dotnet/roslyn/pull/47960/files#diff-78a9e01477cc9984dc175256d51a8fb2R35).